### PR TITLE
Feat/in 3066/trinosetup

### PR DESCRIPTION
**Pull request by hkarira1**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -203,11 +204,56 @@ workflows:
...
+          filters:
+            branches:
+              only:
+                - dev
+      - au-deployment-approval:
+          name: au deployment approval
+          type: approval
+          requires:
+            - Build production trino
+          filters:
```
Comment (vighneshhevo) [inline .circleci/config.yml:222]: Add context here to limit who can approve the deployment

File: .circleci/config.yml [hunk 2]
```diff
@@ -203,11 +204,56 @@ workflows:
...
+            - org-global
+            - env-au
+          filters:
+            branches:
+              only:
+                - dev
+      - us-deployment-approval:
+          name: us deployment approval
+          type: approval
+          requires:
```
Comment (vighneshhevo) [inline .circleci/config.yml:242]: Add context here to limit the approvers scope in production environment

**Review by vighneshhevo — CHANGES_REQUESTED** [review]

**Review by 91pratyush — APPROVED** [review]

**Review by tremendoustj — DISMISSED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (hkarira1) [reply .circleci/config.yml:228]: Added the contexts in approval step as well and putting restrction on the context level

File: .circleci/config.yml (hunk 2 above)
Comment (hkarira1) [reply .circleci/config.yml:248]: Added the contexts in approval step as well and putting restrction on the context level

**Review by vighneshhevo — APPROVED** [review]

**Review by 91pratyush — APPROVED** [review]

**Pull request merged**